src/SpoolmanManager.cpp

```
@@ -1641,11 +1907,21 @@ bool SpoolmanManager::syncSpool(const SpoolmanSyncRequest& req, int& resolvedSpo
     }
 
     int spoolId = findSpoolByUuid(filamentId, req.spool_id);
+    if (spoolId == -2) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)